internal/nodes/validator.go

```diff
@@ -115,6 +115,42 @@ func sslKeyCertHostRulePresent(key, host string) (bool, string) {
 	return false, ""
 }
 
+func destinationCAHTTPRulePresent(key, host, path string) (bool, string) {
+	// from host check if httprule is present
+	found, pathRules := objects.SharedCRDLister().GetFqdnHTTPRulesMapping(host)
+	if !found {
+		utils.AviLog.Debugf("key: %s, msg: Couldn't find fqdn %s to httprule mapping in cache", key, host)
+		return false, ""
+	}
+
+	rule, isValid := pathRules[path]
+	if !isValid {
+		utils.AviLog.Debugf("key: %s, msg: Couldn't find path %s to httprule mapping in cache", key, path)
+		return false, ""
+	}
+
+	ruleNSName := strings.Split(rule, "/")
+	// from hostrule check if hostrule.TLS.SSLKeyCertificate is not null
+	httpRuleObj, err := lib.GetCRDInformers().HTTPRuleInformer.Lister().HTTPRules(ruleNSName[0]).Get(ruleNSName[1])
+	if err != nil {
```

Comment:
Can these two blocks if and else if becoming 1 block of if?

Reply:
This is  just for different logs, i believe the two logs need to be called out separately.